smoke-tests/src/test/groovy/io/opentelemetry/smoketest/SmokeTest.groovy

```diff
@@ -46,7 +46,7 @@ abstract class SmokeTest extends Specification {
 
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper()
 
-  protected static OkHttpClient client = OkHttpUtils.client()
+  protected static final OkHttpClient CLIENT = OkHttpUtils.client()
```

Comment:
Can we instead ignore this rule? I think @trask will object, but why static final variable MUST be in uppercase?

Reply:
spotBugs actually doesn't complain about the uppercase, just the `final` declaration. As for the uppercase, I believe it's  a JLS convention for constants, but given the naming exception done for loggers, I don't see why it cannot be the same here - just possibly, have to look out for other static analysis checkers complaining about it ;)

Reply:
I don't think JLS has any naming conventions. Has it?

Reply:
https://docs.oracle.com/javase/specs/jls/se11/html/jls-6.html look for "Constant Names" section.
Emphasis, mine.

>The names of constants in interface types should be, and final variables of class types **may** conventionally be, a sequence of one or more words, acronyms, or abbreviations, all uppercase


The capitalisation could have been a misrepresentation though, and the intent may have been to use them for primitives, but it somehow stuck for everything

Reply:
> Can we instead ignore this rule? I think @trask will object, but why static final variable MUST be in uppercase?

I do not object 😂

static final variables should only be uppercase if they are constants, see https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names for "what is a constant"